### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by agokarn**
This change contains the attested OHTTP client code for Confidential inferencing https://techcommunity.microsoft.com/blog/azure-ai-services-blog/azure-ai-confidential-inferencing-preview/4248181

### CODE REVIEW

File: ohttp-client/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+serde_json = "1.0"
+tracing = "0.1"
+tracing-subscriber = { version = "0.3.18", features = ["default", "json", "env-filter"] }
+infer = "0.16.0"
+
+[dependencies.verifier]
+path= "../verifier"
+
+[dependencies.bhttp]
+git = "https://github.com/kapilvgit/ohttp.git"
```
Comment (kapilvgit) [inline ohttp-client/Cargo.toml]: Change this to microsoft/ohttp

File: Cargo.lock
Comment (kapilvgit) [inline Cargo.lock:1]: This file should not be checked in.

File: ohttp-client/README.md
Comment (kapilvgit) [inline ohttp-client/README.md:1]: I think this README moves one level up, and should include a section on Contributing and Trademarks

File: ohttp-client/src/main.rs
Comment (kapilvgit) [inline ohttp-client/src/main.rs:1]: Update license.

File: verifier/src/err.rs
Comment (kapilvgit) [inline verifier/src/err.rs:1]: Update license

File: verifier/src/lib.rs
Comment (kapilvgit) [inline verifier/src/lib.rs:1]: Update license.

File: ohttp-client/README.md
Comment (agokarn) [reply ohttp-client/README.md:1]: it had moved up but was here as well. thanks. We can address  Contributing and Trademarks later

**Review by kapilvgit — APPROVED** [review]

**Pull request merged**